install.sh

```diff
@@ -26,8 +26,11 @@ function build_and_install_beta() {
   ./build.sh >/dev/null
   cd ..
 
+  local lastest_commit
+  lastest_commit=$(git rev-parse --short=8 HEAD);
+
   local beta_version
-  beta_version='(non-stable) beta after '"$LATEST_BASHUNIT_VERSION"' ['"$(date +'%Y-%m-%d')"']'
+  beta_version='(non-stable) beta after '"$LATEST_BASHUNIT_VERSION"' ['"$(date +'%Y-%m-%d')"'] 🐍 $lastest_commit'
```

Comment:
Missing quotes on the new variable `'"$lastest_commit"`
Explanation: variables cannot be rendered inside single quotes, so you first need to close the singe quote and then concat the variable wrapped by double quote to avoid weird bash-behaviour